app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def full_name
```

Comment:
where does this method belong? where are we using it? what's it's purpose?

currently, all models have access to this - is this what we want?